### PR TITLE
Update API calls in NOM visualizations to reflect Berkeley schema

### PR DESCRIPTION
**Pull request by samobermiller**
PR updating API calls in NOM visualizations to Berkeley schema. https://github.com/microbiomedata/issues/issues/726

## All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/microbiomedata/nmdc_notebooks/blob/main/.github/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/microbiomedata/nmdc_notebooks/pulls) for the same update/change?
* [x] Does your PR link to an issue?
* [x] Have you described the changes this PR will make?

### Notebook Fix Submissions:

* [ ] Does your PR include links to the updated notebook **(in the branch)** for review using [nbviewer](https://nbviewer.jupyter.org/), [Colab](https://colab.research.google.com/), and [reviewnb](https://www.reviewnb.com/)? These three are the preferred ways to review changes and additions to notebooks during review.

### CODE REVIEW

File: NOM_visualizations/python/nom_data.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   },
```
Comment (kheal) [inline NOM_visualizations/python/nom_data.ipynb:10]: <pre class="ql-syntax" spellcheck="false">Line #1.    omicsprocess_dataobj=func.get_id_results(
</pre><p>Change the name of the python objects to remove "omicsprocessing" please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/80/discussion/#comment-1783145826'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by kheal** [comment]
Nice job @samobermiller  and @bmeluch.  I found a few references to "omics processing" that should be "data generation", but the functionality looks great.

File: NOM_visualizations/python/nom_data.ipynb (hunk 1 above)
Comment (brynnz22) [inline NOM_visualizations/python/nom_data.ipynb:10]: <p>Change "Find the omics processing records" to "Find the data generation records" Also, change the variables from <code>omics_processing_id</code> <code>omicsprocess_has_output</code> <code>omicsprocess_has_input</code> to allude to data generation.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (brynnz22) [inline NOM_visualizations/python/nom_data.ipynb:10]: <p>Change "omics processing" to "data generation" And change variable (mentioned in comment above too) for <code>omicsprocess_has_input</code> to allude to data generation instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by samobermiller** [comment]
These make sense! I guess in the back of my head I was thinking theyre still technically omics records, but I see thats its confusing cause we query data generation

**Comment by brynnz22** [comment]
I think Katherine and I must have reviewed at the same time. I had similar comments.

**Review by kheal — APPROVED** [review]

**Pull request merged**